transmogrifier/sources/transformer.py

```diff
@@ -72,7 +70,20 @@ def __next__(self) -> TimdexRecord:
                 continue
 
     @final
-    def write_timdex_records_to_json(self, output_file: str) -> int:
+    def _write_output_files(self, output_file: str) -> None:
```

Comment:
If this method is turned into a public method, I think a docstring here would be nice to also inform the reader that this method is also driving the iteration through all input records, and transformation of them, via the `while records:...` bit.